### PR TITLE
Create a ref doc for scheduled jobs

### PR DESCRIPTION
**Pull request by janetkuo**
Supersedes #1111, since this doc needs to be merged into 1.4 branch (instead of master). All comments in #1111 are addressed. 

Ref kubernetes/kubernetes#28496, kubernetes/features#19

@soltysh @erictune @jaredbhatti @kubernetes/docs

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1123)

### CODE REVIEW

**Comment by janetkuo** [comment]
PTAL

**Comment by erictune** [comment]
LGTM

File: docs/user-guide/scheduled-jobs.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+### Prerequisites
+
+You need a working Kubernetes cluster at version >= 1.4, with batch/v2alpha1 API turned on by passing
+`--runtime-config=batch/v2alpha1` while bringing up the API server (see [Turn on or off an API version
+for your cluster](/docs/admin/cluster-management/#turn-on-or-off-an-api-version-for-your-cluster) for
+more). You cannot use Scheduled Jobs on a hosted Kubernetes provider that has disabled alpha resources.
+
+## Creating a Scheduled Job
+
+Here is an example Scheduled Job. Every minute, it runs a simple job to compute π to 2000 places and prints it out.
```
Comment (soltysh) [inline docs/user-guide/scheduled-jobs.md]: The sample job prints date and `Hello from the Kubernetes cluster` not computes PI.

File: docs/user-guide/scheduled-jobs.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+$ kubectl delete jobs hello-1201907962 hello-1202039034 ...
+job "hello-1201907962" deleted
+job "hello-1202039034" deleted
+...
+```
+
+Once the jobs are deleted, the pods created by them are deleted as well. Note that all jobs created by scheduled
+job "hello" will be prefixed "hello-". You can delete them at once with `kubectl delete jobs --all`, if you want to 
+delete all jobs in the system (not just the ones created by "hello".)
```
Comment (soltysh) [inline docs/user-guide/scheduled-jobs.md]: That should remove all in the current namespace, not in the entire system.

File: docs/user-guide/scheduled-jobs.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+* `Forbid`: forbids concurrent runs, skipping next run if previous hasn't finished yet
+* `Replace`: cancels currently running job and replaces it with a new one
+
+Note that concurrency policy only applies to the jobs created by the same scheduled job. If there are multiple
+scheduled jobs, their respective jobs are always allowed to run concurrently.
+
+### Suspend
+
+The `.spec.suspend` field is also optional. If set to `true`, all subsequent executions will be suspended. It does not 
+apply to already started executions. Defaults to false.
```
Comment (soltysh) [inline docs/user-guide/scheduled-jobs.md]: I'd add a not that all modifications to SJ (esp. the `.spec`) part are applied only to the next run.

**Comment by soltysh** [comment]
Since I have a few comments, I allow myself to remove Tech Review approval label.

**Comment by erictune** [comment]
Good eye, thanks Maciej

File: docs/user-guide/scheduled-jobs.md (hunk 1 above)
Comment (janetkuo) [inline docs/user-guide/scheduled-jobs.md]: Nice catch, fixed, thanks

File: docs/user-guide/scheduled-jobs.md (hunk 2 above)
Comment (janetkuo) [inline docs/user-guide/scheduled-jobs.md]: fixed

File: docs/user-guide/scheduled-jobs.md (hunk 3 above)
Comment (janetkuo) [inline docs/user-guide/scheduled-jobs.md]: Note added https://github.com/kubernetes/kubernetes.github.io/pull/1123/files#diff-28c2ccf1cca081dae5a6953354170037R146

**Comment by janetkuo** [comment]
Thanks! All comments addressed, PTAL

**Comment by soltysh** [comment]
LGTM

**Comment by janetkuo** [comment]
Looks like we don't separate Tech LGTM and Docs LGTM anymore. So manually merging.

**Pull request merged**